apps/src/templates/sectionProgress/progressTables/ProgressTableLevelIcon.jsx

```diff
@@ -16,23 +18,47 @@ const styles = {
     textAlign: 'center',
     color: color.charcoal,
     fontSize: 20
+  },
+  unpluggedPlaceholderContainer: {
+    height: 0,
+    opacity: 0
   }
 };
 
-function LevelIcon({icon}) {
+const placeholderUnpluggedBubble = (
+  <div style={styles.unpluggedPlaceholderContainer}>
+    <div>
+      <ProgressTableLevelBubble
+        levelStatus={LevelStatus.not_tried}
+        unplugged={true}
+        disabled={true}
+        url=""
+      />
+    </div>
+  </div>
+);
+
+function LevelIcon({icon, unplugged}) {
+  // The width of our unplugged bubble depends on the localization of the text,
+  // so to allow that to be determined at render time, we don't set an explicit
+  // width for unplugged bubbles and instead render an invisible one behind the
+  // icon to determine its width.
```

Comment:
I could use some clarification here, are we rendering the hidden progress bubble here because the header is determining the width of the column?

Reply:
more specifically, we're adding the hidden progress bubble here so that the intrinsic content width of the unplugged icon will match that of the bubble, regardless of context.

hopefully the comment i just added to `ProgressTableContentView.columnWidthStyle` will clarify why this is necessary -- the content width of the `ProgressTableLevelBubble`s determine the width of the body columns, but for our header columns to have the same width without it being provided explicitly, they need to have the same content width as the body columns. so in our case, this is determining the width of the header columns, but specifically _not_ of the body columns (`ProgressTableLessonNumber` sizes itself based on its container, so does not contribute to the determination of the header column width). lmk if that's still unclear!